web/src/test/java/com/synopsys/integration/alert/web/api/home/HomeActionsTest.java

```diff
@@ -23,30 +23,30 @@
 import org.springframework.security.web.csrf.HttpSessionCsrfTokenRepository;
 
 import com.synopsys.integration.alert.common.action.ActionResponse;
-import com.synopsys.integration.alert.web.security.authentication.saml.SAMLContext;
+import com.synopsys.integration.alert.component.authentication.security.saml.SAMLContext;
 
-public class HomActionsTest {
+public class HomeActionsTest {
     @Test
     public void testVerifyAuthenticationValid() {
+        HttpServletRequest servletRequest = new MockHttpServletRequest();
+        HttpServletResponse servletResponse = new MockHttpServletResponse();
+        HttpSessionCsrfTokenRepository csrfTokenRepository = new HttpSessionCsrfTokenRepository();
```

Comment:
These may be better created in a beforeEach method? Looks like the majority of these classes are used in most tests

Reply:
Encapsulated the initialization of the fields.